docs: added registered trademark symbol to mentions of external software (#19684)
druid-io/druid: docs/development/extensions-contrib/druid-ranger-security.md

  ~ under the License.
  -->

> This Apache Druid extension adds an Authorizer which implements access control for Druid, backed by [Apache&circledR; Ranger](https://ranger.apache.org/). Please see [Authentication and Authorization](../../operations/auth.md) for more information on the basic facilities this extension provides.

Reviewer: ```suggestion
This Apache&circledR; Druid extension adds an Authorizer which implements access control for Druid using [Apache&circledR; Ranger](https://ranger.apache.org/). See [Authentication and authorization](../../operations/auth.md) for more information.
```
